s2i/README.md

```diff
@@ -13,15 +13,18 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/s2i/s
 
 ### Parameters
 
-* **BUILDER_IMAGE**: The name of the image containing the s2i command. See
+* **BUILDER_IMAGE**: The location of the s2i builder image.
```

Comment:
Don't think this param is needed. It has to always be the same image

Reply:
you mean the `S2I_IMAGE` image right (the one that has the `s2i` binary in) ?